src/kbmod/reprojection.py

```diff
@@ -287,29 +281,26 @@ def _reproject_work_unit_in_parallel(
     # when all the multiprocessing has finished, convert the returned numpy arrays to RawImages.
     concurrent.futures.wait(future_reprojections, return_when=concurrent.futures.ALL_COMPLETED)
     time_stamps = []
-    image_list = []
+    stack = ImageStack([])
     for result in future_reprojections:
         science_add, variance_add, mask_add, time = result.result()
-        science_raw_image = RawImage(img=science_add, obs_time=time)
-        variance_raw_image = RawImage(img=variance_add, obs_time=time)
-        mask_raw_image = RawImage(img=mask_add, obs_time=time)
-
         psf = _get_first_psf_at_time(work_unit, time)
+
         # And then stack the RawImages into a LayeredImage.
         new_layered_image = LayeredImage(
-            science_raw_image,
-            variance_raw_image,
-            mask_raw_image,
+            science_add,
+            variance_add,
+            mask_add,
             psf,
+            time,
         )
 
         # append timestamps and layeredImages to lists
         time_stamps.append(time)
```

Comment:
And can remove this line for the same reason as above.

Reply:
cleaned this up too.